contracts/MeshToken.sol

```diff
@@ -0,0 +1,51 @@
+pragma solidity ^0.4.15;
+
+import 'zeppelin-solidity/contracts/token/CappedToken.sol';
+
+contract MeshToken is CappedToken {
+  string public name = "MESH TOKEN";
+  string public symbol = "MESH";
+  uint8 public decimals = 18;
+
+  /**
+   *  @dev adress that is allowed to mint new token
+   */
+  address public minter;
+
+  /**
+   * @dev constructor for mesh token
+   * @param _cap The total number of tokens that can ever be minted
+   */
+  function MeshToken(uint256 _cap) CappedToken(_cap) public {}
```

Comment:
Do you need to pass in tokenAddress as well? CrowdSale is minting the token with an address instead of a cap.